console/src/components/Services/Data/TableBrowseRows/EditActions.js

```diff
@@ -103,14 +144,19 @@ const editItem = (tableName, colValues) => {
         error: { message: errorMessage },
       });
     }
-
+    let returning = [];
+    if (isMigration) {
+      returning = columns.map(col => col.column_name);
+    }
+    const { pkClause, oldItem } = state.tables.update;
     const reqBody = {
       type: 'update',
       args: {
         table: tableDef,
         $set: _setObject,
         $default: _defaultArray,
-        where: state.tables.update.pkClause,
+        where: pkClause,
```

Comment:
Do not use `pkClause` for this. Write a function to change the elements of `oldItem` to a where clause. 
The function can be defined like so:

```javascript
const makeWhereClause = (oldItem) => {
     const listOfElems = [];
     const oldItemKeys = Object.keys(oldItem);
     oldItemKeys.forEach(key => {
            listOfElems.push({ [key]: { '$eq': oldItem[key] } }) 
      });
     
     return {
         '$and': listOfElems,
     };
}
```